plugin/go.mod

```diff
@@ -12,4 +12,6 @@ require (
 	google.golang.org/grpc v1.31.0
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v0.0.0-20200806223615-7419b444ee17 // indirect
 	google.golang.org/protobuf v1.25.0
+	github.com/sclevine/spec v1.4.0
+    github.com/stretchr/testify v1.5.1
```

Comment:
Use tabs instead of spaces.  (You should probably `gofmt` on every save)

Reply:
It's weird, it displays correctly on my local machine. Tried to fix it.

Reply:
Still not fixed.  Is your machine expanding tabs to a width of 4 spaces?  If so, it would look fine to you.  `gofmt`

Reply:
It seems so, it just replaces 1 tab as 4 spaces. So, fixed it, now it looks here as it's supposed to.